packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -158,7 +158,8 @@ interface GetVersionsTelemetryProps {
     cacheEntryAge?: number;
     cacheSummarizerExpired?: boolean;
 }
-export class OdspDocumentStorageService implements IDocumentStorageService {
+
+export abstract class OdspDocumentStorageServiceBase implements IDocumentStorageService {
```

Comment:
Let's put this class in its own file of same name: OdspDocumentStorageServiceBase.ts